### PR TITLE
Nomination: Iulian Dragos

### PR DESCRIPTION
**Pull request by fommil**
// @dragos @rorygraves

### CODE REVIEW

**Comment by propensive** [comment]
Thanks, @fommil. Can I get a quick message of confirmation (or review) from @rorygraves and @dragos, please?

**Comment by rorygraves** [comment]
Seconded

File: nominations/iulian-dragos.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+This candidate has been nominated by @fommil and seconded by @rorygraves.
+
+## Qualifications and Experience
+
+- expert on the scala interactive compiler
+- wrote the first scala LSP implementation
+
+## Affiliations
+
+- standing as an individual
```
Comment (propensive) [inline nominations/iulian-dragos.md]: This will need to indicate Triplequote, for transparency.

**Comment by dragos** [comment]
Thanks for the nomination, guys! I'll be happy to serve.

File: nominations/iulian-dragos.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+## Working Groups
+
+This candidate is nominated as a candidate to the LSP Working Group (LSP-WG).
```
Comment (propensive) [inline nominations/iulian-dragos.md]: @dragos may modify which of the two groups he feels best representing. If in doubt, include both.

File: nominations/iulian-dragos.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+## Proposition
+
+See "Qualifications and Experience".
```
Comment (propensive) [inline nominations/iulian-dragos.md:22]: This is for @dragos to complete, as he sees fit. :)

**Comment by propensive** [comment]
Thanks, @dragos! I've merged the PR, but feel free to raise a new one with more details of your "proposition" in the final section, or to add the STP-WG too.

**Pull request merged**